### ISSUE TITLE
Iterated ensemble smoother gui pane says number of iterations: 1 but runs iteration 0 and 1

### ISSUE THREAD

**Issue by eivindjahren** [equinor/ert#4841]
**Describe the bug**
number of iterations = 1 seems like it would run just iteration 1

**To reproduce**
Steps to reproduce the behaviour:
1. `pip install ert`
2. `ert gui my_config.ert`
3. choose iterated ensemble smoother in simulation mode 
4. select 4 iterations
5. press run

**Expected behaviour**
It runs iterations 0-4 (ie. 5 iterations ran)

**Screenshots**
![image](https://user-images.githubusercontent.com/32731672/217801515-0aa87cc1-e6a9-47ef-a271-6c944348e9ff.png)

**Environment**
 - OS: Any
 - ERT: 4.3
 - Python version
 - Remote/HPC execution involved: no

**Comment by eivindjahren** [comment]
Maybe rename to number of analysis steps

**Comment by oyvindeide** [comment]
Not really a bug as it is intended as number of iterations of the algorithm, but could be clarified.

**Comment by eivindjahren** [comment]
Maybe reworded to "number of analysis iterations" ?

**Comment by eivindjahren** [comment]
Rename may be superflous
